content/blog/2020/2020-04-20-wdl-head/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Win-Draw-Loss evaluation"
+slug: "wdl-head"
+published: 2020-04-20
+author: "crem"
+# image: "name"
+# cover: "cover.png" # Default
+loadChessbaseScripts: true
+---
+
+Traditionally, computer chess engines evaluated position in [pawns](https://www.chessprogramming.org/Centipawns).
+This is a convenient choice, as material is a foundation of position evaluation, which is true also among humans.
+Situations where importance of positional aspect strongly outweigh material are often thought as beautiful exceptions rather than a rule.
+
+It is however not the most logical way to score a position.
+Using pawns sounds arbitrary, and it's also not linear and the same delta has different meanings on a scale:
+while going from 0 to 1 pawn advantage keeps the position quite holdable, going from 1 to 2 usually makes it mostly lost.
+Going from 10 to 11 makes no difference at all.
+
+<!--more-->
+
+## Using probabilities
+
+What other values can be used as an evaluation of the position?
+The natural choice appears to be using of **probability of winning**.
+
+Probability of winning is the evaluation method that DeepMind chosen for their AlphaGo and later AlphaZero engines.
+It's also what LCZero used up to July 2019.
+
+Internally, LCZero has always used probabilities, but as chess engine protocol UCI requires value in pawns, it just converted it's output using the following function:
```

Comment:
but as **the UCI** chess engine protocol ~~UCI~~ requires **a** value
converted  ~~it's~~**its** output